docs/platform/what-is-tizen/overview.md

```diff
@@ -48,6 +48,37 @@ The most important strengths that enable Tizen to move to the next level are IoT
   ![IoTivity for Connectivity](media/about_tizen_6.png)
 
 
+## AI and Machine Learning
+### On-Device & Among-Device AI Pipelines for Inference
+
+NNStreamer (https://github.com/nnstreamer/nnstreamer) is a set of GStreamer plugins that allow GStreamer developers to adopt neural network models easily and efficiently and neural network developers to manage neural network pipelines and their filters easily and efficiently.
+
+- Provide neural network framework connectivities (e.g., tensorflow, caffe) for GStreamer streams.
+  - Efficient Streaming for AI Projects: Apply efficient and flexible stream pipeline to neural networks.
+  - Intelligent Media Filters: Use a neural network model as a media filter / converter.
+  - Composite Models: Multiple neural network models in a single stream pipeline instance.
+  - Multi Modal Intelligence: Multiple sources and stream paths for neural network models.
+- Provide easy methods to construct media streams with neural network models using the de-facto-standard media stream framework, GStreamer.
+  - GStreamer users: use neural network models as if they are yet another media filters.
```

Comment:
To maintain consistency with other bullet points, “Users” and “Use” should be capitalized. The word “filter” should be singular here. 
```suggestion
  - GStreamer Users: Use neural network models as if they are yet another media filter.
```

Reply:
Updated!